### PR TITLE
update hl_sys_time on Windows

### PR DESCRIPTION
**Pull request by ConstNW**
#280

### CODE REVIEW

File: src/std/sys.c [hunk 1]
```diff
@@ -163,14 +163,18 @@ HL_PRIM void hl_sys_exit( int code ) {
 
 HL_PRIM double hl_sys_time() {
 #ifdef HL_WIN
+	#define EPOCH_DIFF	(134774*24*60*60.0)
```
Comment (Aurel300) [inline src/std/sys.c]: What is this value based on?

Comment (ConstNW) [reply src/std/sys.c]: it's epoch diff between Unix epoch and Windows FILETIME epoch (started January 1, 1601 (UTC))

**Comment by ncannasse** [comment]
This new sys_time is 1.6 slower than previous implementation but I guess it's good enough.
Please note that there also might be some non linear timings, as explained here https://stackoverflow.com/questions/34557407/queryperformancecounter-or-getsystemtimepreciseasfiletime-when-using-sntp

**Comment by Shoozza** [comment]
Btw "GetSystemTimePreciseAsFileTime" is only available on Windows 8+. So this patch removed support for earlier versions of Windows.

https://docs.microsoft.com/en-us/windows/win32/api/sysinfoapi/nf-sysinfoapi-getsystemtimepreciseasfiletime

**Pull request merged**